### PR TITLE
Make Messages submodule of CommandParser

### PR DESCRIPTION
**Pull request by panpawn**
I don't think this should have it's own global... Besides, this makes it hotpatchable with /hotpatch chat

### CODE REVIEW

**Comment by panpawn** [comment]
Wait, this isn't done

**Comment by Zarel** [comment]
This works until the next time I want to export something from `Messages`, which is slated to become a fair bit fancier than it currently is.

Maybe just attach it to `CommandParser.Messages`

**Comment by panpawn** [comment]
So, to clarify, in CommandParser:
`exports.Messages = require('./messages);`

Then, in the PM command:
`CommandParser.Messages.send...`

Correct?

**Comment by panpawn** [comment]
Well, I updated this based off of that assumption.  :P

**Comment by Zarel** [comment]
Yes, that's what I wanted, thanks.

File: messages.js [hunk 1]
```diff
@@ -9,9 +9,7 @@
 
 'use strict';
 
-let Messages = module.exports;
-
-Messages.send = function (target, context) {
+exports.send = function (target, context) {
```
Comment (Zarel) [inline messages.js]: Can you revert these changes, too?

Comment (panpawn) [inline messages.js]: Sure

Comment (panpawn) [inline messages.js]: Done

**Comment by panpawn** [comment]
Updated 😃
